src/components/Footer.js

```diff
@@ -1,72 +1,96 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect, Suspense } from 'react'
+import { useTranslation } from 'react-i18next'
```

Comment:
We have replaced this with `gatsby-plugin-react-i18next`. Do you mind updating it please 
```
import {  useTranslation } from 'gatsby-plugin-react-i18next'
```

Reply:
okay on it, thanks